.changeset/brown-elephants-boil.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/wallets": patch
+---
+
+ConnectWallet v3 updates
```

Comment:
Can we remove this file and keep the other one?

Reply:
sure